### DISCUSSION TITLE
[Android] add image to app

### DISCUSSION THREAD

**Discussion by buildsummer** [beeware/toga#1355] [Q&A] [answered]
Hi,

I need to add image on my app. And if user clicks some button, it should be changed to another image.

On Windows, toga.Imageview works well. But after install app in my phone using apk, it does not work.

I saw Imageview does not support android on toga tutorial pdf file.

Is there any way to add image that can be changed by responsing some action like clicking button on android app?

**Comment by freakboy3742** [comment, marked as answer]
Yes, it is possible; and the code you need shouldn't be any different to what you're doing on Windows. The critical thing will be to get the path specifying the path to the image file correct. 

The examples folder gives an [example of an ImageView](https://github.com/beeware/toga/blob/master/examples/imageview/imageview/app.py) that works across platforms.

**Reply by buildsummer** [reply]
Thank you!

I found the problem of my code after seeing examples you mentioned. And I fixed the problem. My app works fine.

And I have one more question.

Is there any why to rotate screen on android?

**Reply by freakboy3742** [reply]
It *can* be done; however, it's not currently exposed as a feature you can easily modify in Python code. 

You can do it manually by adding `android:screenOrientation="fullSensor"` to the `<activity>` tag in your AndroidManifest.xml. You'll need to do this every time you rebuild your app. You could also build a custom Android briefcase template to include this change.

**Reply by buildsummer** [reply]
Thank you! It works well.
